conf/defaults.ini

```diff
@@ -887,6 +887,12 @@ allow_loading_unsigned_plugins =
 plugin_admin_enabled = false
 plugin_catalog_url = https://grafana.com/grafana/plugins/
 
+#################################### Grafana Live ##########################################
+[live]
+# max_connections to Grafana Live WebSocket endpoint per Grafana server instance. See Grafana Live docs if you are
+# going to make it higher than default 100 since this can require some OS and infrastructure tuning. -1 for unlimited.
```

Comment:
we should also say that 0 will disable clients connecting?

Reply:
done